### ISSUE TITLE
Ontology <> `@Context` File

### ISSUE THREAD

**Issue by bumblefudge** [swicg/potential-charters#5]
A [question was raised on the list](https://lists.w3.org/Archives/Public/public-swicg/2024Oct/0001.html) as to whether a definitive [RDF?] Ontology would be in scope of an AP-updating WG, however scoped (maintenance or net-new, AP-only or Social-Web-Wide). Similar points have been made on SocialHub by implementers not active here.

Personally, I feel like this would be a great idea, but as with any question of prioritization, _committing to maintaining_ an ontology is a good bit of work and I hesitate to make it a _blocking_ deliverable.  Is there a way to make it a "stretch goal" or "potentially in-scope" deliverable at time of chartering? Would putting it in scope make it easier to find a grant or organization to "sponsor" the work, which I assume to be significant regardless of the format/toolchain chosen to do it in?

**Comment by cristianolongoodhl** [comment]
Ideally, this work should proceed "in parallel" with specification, so that the OWL modelling would provide a formal definition (where it is possible, considering the limitations and the scope of OWL) and validation of human-readable normative specifications.

**Comment by evanp** [comment]
It's in scope for the CG charter, so I think that's plenty?
